src/views/home/components/token-list/token-list.view.tsx

```diff
@@ -83,50 +84,57 @@ export const TokenList: FC<TokenListProps> = ({
     setCustomToken({ status: "pending" });
 
     if (ethersUtils.isAddress(searchTerm) && newFilteredTokens.length === 0) {
-      setCustomToken({ status: "loading" });
+      if (TOKEN_BLACKLIST.includes(searchTerm)) {
+        setCustomToken({
+          error: "We don't support this token by the moment.",
```

Comment:
I think "by the moment" is better worded as "at the moment", I would though change this text to: "We currently do not support this token."

Reply:
I prefer the first one, updated.